### ISSUE TITLE
Internalise `subset_clusters_by...`

### ISSUE THREAD

**Issue by thibautjombart** [reconhub/epicontacts#47]
These should be put as options in `subset` and internalized (i.e. no export, no dedicated doc).

User cases should still be illustrated in the doc of `subset`

**Comment by nistara** [comment]
Not sure I understand this...by internalize, do you mean give and options for `id` or `size` within the same `subset` function? Or include this as an option with the base R `subset` function? Something like what you did for plot? Thank you for explaining further!

**Comment by thibautjombart** [comment]
Hey! 
I mean: keep the function as an internal one, without exporting it, and call it from the 'subset' method for epicontacts data, which will look like, eg:
`subset(x, cluster_size=...)`

**Comment by finlaycampbell** [comment]
Woops, accidentally closed the issue - I've made a first attempt at internalising the functions, and have submitted a pull request.  I've internalised the subset_cluster_by_id and subset_cluster_by_size into the subset function. If node/edge attributes are provided and the epicontacts object is subset by cluster, the linelist/contacts is subset by node/edge attributes first, and the clusters identified from this subsetted epicontacts object. Nistara's documentation and examples have been carried over, the vignette hasn't been updated though.
